### PR TITLE
KAFKA-16223: Replace EasyMock/PowerMock with Mockito for KafkaConfigBackingStoreTest

### PR DESCRIPTION
**Pull request by FrankYang0529**
### Committer Checklist (excluded from commit message)
- [ ] Verify design and implementation 
- [ ] Verify test coverage and CI build status
- [ ] Verify documentation (including upgrade notes)

### CODE REVIEW

File: connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaConfigBackingStoreTest.java [hunk 1]
```diff
@@ -960,6 +666,14 @@ private void expectConvertWriteRead(final String configKey, final Schema valueSc
                 });
     }
 
+    private void expectConvertWriteAndRead(final String configKey, final Schema valueSchema, final byte[] serialized,
```
Comment (chia7712) [inline connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaConfigBackingStoreTest.java]: please remove this unused function

Comment (FrankYang0529) [reply connect/runtime/src/test/java/org/apache/kafka/connect/storage/KafkaConfigBackingStoreTest.java]: Removed it. Thanks.

**Comment by chia7712** [comment]
@FrankYang0529 Could you please rebase code to trigger QA ?

**Comment by gharris1727** [comment]
@chia7712 I triggered a rebuild. You can do it to if you login to the underlying Jenkins: https://issues.apache.org/jira/browse/INFRA-25442

**Comment by chia7712** [comment]
>  triggered a rebuild. You can do it to if you login to the underlying Jenkins: https://issues.apache.org/jira/browse/INFRA-25442

Big thanks!

**Review by chia7712 — APPROVED** [review]
LGTM

**Pull request merged**